src/Tribe/Status/Manager.php

```diff
@@ -190,15 +164,17 @@ public function get_trigger_statuses( $commerce ) {
 
 		$trigger_statuses = [];
 
-		$commerce = $this->check_for_full_provider_name( $commerce );
+		$commerce = $this->get_provider_slug( $commerce );
 
 		if ( ! isset( $this->statuses[ $commerce ]->statuses ) ) {
 			return $trigger_statuses;
 		}
 
-		$filtered_statuses = wp_list_filter( $this->statuses[ $commerce ]->statuses, [
-			'trigger_option' => true,
-		] );
+		$filtered_statuses = wp_list_filter(
+			$this->statuses[ $commerce ]->statuses, [
```

Comment:
:no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 0e9f99a45667a24b3a63c0884170b599</sub>

Reply:
fixed